src/Pelagos/Entity/DatasetSubmission.php

```diff
@@ -3,7 +3,7 @@
 namespace Pelagos\Entity;
 
 use Doctrine\ORM\Mapping as ORM;
-
+use Pelagos\Util\PelagosUtilities;
```

Comment:
Do you want to move this use statement below, for better grouping?

Reply:
yeah, we normally alphabetize them.